.github/workflows/test.yml

```diff
@@ -126,9 +126,9 @@ jobs:
         target_url: "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
       env:
         GITHUB_TOKEN: ${{ secrets.CI_TOKEN }}
-      if: always() && ${{ needs.build.result != 'cancelled' }} && ${{ github.event_name == 'repository_dispatch' }}
+      if: ${{ !cancelled() && github.event_name == 'repository_dispatch' }}
```

Comment:
Was wandering whether !cancelled works even in case of failures

Reply:
Yes, in my tests the step is executed even in presence of a previously failed step. Ex: https://github.com/palexster/liqo/runs/3328516435?check_suite_focus=true